statistics/handle/cache/internal/lfu/lfu_cache.go

```diff
@@ -77,6 +79,11 @@ func (s *LFU) Get(tid int64, _ bool) (*statistics.Table, bool) {
 
 // Put implements statsCacheInner
 func (s *LFU) Put(tblID int64, tbl *statistics.Table) bool {
+	if tbl == nil {
+		logutil.BgLogger().Fatal("table is nil", zap.Int64("tableID", tblID))
+	} else {
+		logutil.BgLogger().Info("table", zap.Int64("tableID", tblID), zap.Stack("stack"))
```

Comment:
This log may cause some performance issues, better to remove this log.

Reply:
removed.